### DISCUSSION TITLE
popup vanishes after click/interaction on MacOS 15

### DISCUSSION THREAD

**Discussion by moritzWa** [MonitorControl/MonitorControl#1607] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

it only shows up in the menu bar if I search the app and open it but then vanishes if I try to change the brightness  
see gif
![2024-09-18 13 02 23](https://github.com/user-attachments/assets/e91d0687-a06b-48f3-b2ad-a4711b53ea81)
it doesn't happen if I click on preferences/settings

### Steps to reproduce

1. check menu bar => app not there
2. use spotlight search and press enter to open app => now app shows up 
3. try to change the brightness of a monitor
4. monitorcontrol popup vanishes

### Expected behavior

sholud not close

### Environment Information (please complete the following information)

```markdown
- macOS version: Sequoia 15.0
- Mac model: MacBook Pro (14-inch, 2021)
- MonitorControl version: Version 4.2.0 Build 7048
- Monitor(s): LG UltraFine, Color LCD | LG UltraFine
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment, marked as answer]
The current 4.2.0 version is not compatible with Sequoia. Please download the unofficial 4.3.x version:

https://github.com/MonitorControl/MonitorControl/issues/1570

Hopefully the situation will be rectified sooner or later.
